backend/app/assets/javascripts/spree/backend/orders/models.js

```diff
@@ -0,0 +1,48 @@
+//= require spree/backend/routes
+
+Spree.Order || (Spree.Order = {});
+
+
+Spree.Order.LineItemModel = Backbone.Model.extend({
+  initialize: function(options) {
+    this.order = this.collection.parent;
+  }
+})
+
+Spree.Order.LineItemCollection = Backbone.Collection.extend({
+  model: Spree.Order.LineItemModel,
+
+  url: function () {
+    return this.parent.url() + "/line_items";
+  },
+})
+
+Spree.Order.OrderModel = Backbone.Model.extend({
+  urlRoot: Spree.routes.orders_api,
+  idAttribute: "number",
+
+  relations: {
+    "line_items": Spree.Order.LineItemCollection,
+    "shipments": Backbone.Collection
+  },
+
+  advance: function(opts) {
+    var options = {
+      url: Spree.routes.checkouts_api + "/" + this.id + "/advance",
+      type: 'PUT',
+    };
+    _.extend(options, opts);
+    return this.fetch(options)
+  }
+});
+
+Spree.Order.OrderModel.fetch = function(number, opts) {
+  var options = (opts || {});
+  var model = new Spree.Order.OrderModel({
+    number: number,
+    line_items: [],
+    shipments: []
+  });
+  model.fetch(options);
+  return model;
+}
```

Comment:
Maybe we should split this file into files for each model? So devs could override individual models (the order for instance)? Not sure though, if this could ever be the case.